engine/src/test/resources/org/hibernate/validator/test/internal/xml/containerelementtype/getter-canDeclareContainerElementTypeConstraintsForListContainingArrayType-mapping.xml

```diff
@@ -1,3 +1,9 @@
+<!--
```

Comment:
I did add those as their absence was causing the build to fail. But we can just add an exclude rule for them, if it's preferred not to have headers here. Just say how it should be :)